### PR TITLE
allow to load YAML configuration files on Windows

### PR DESCRIPTION
**Pull request by fernandezvara**
The function `loadDataFrom` does not take care of Windows users since any of the conditions met and it returns an error.

The change looks for the runtime where it's running and then if the filePath contains a `\`.

### CODE REVIEW

File: altsrc/yaml_file_loader.go [hunk 1]
```diff
@@ -78,6 +80,11 @@ func loadDataFrom(filePath string) ([]byte, error) {
 			return nil, fmt.Errorf("Cannot read from file: '%s' because it does not exist.", filePath)
 		}
 		return ioutil.ReadFile(filePath)
+	} else if runtime.GOOS == "windows" && strings.Contains(u.String(), "\\") {
```
Comment (jszwedko) [inline altsrc/yaml_file_loader.go]: Hmm, don't you want this above the previous clause? Otherwise it'll fall into the `u.Path !+ ""` condition.

Comment (fernandezvara) [inline altsrc/yaml_file_loader.go]: At least on the machines I could try, `u.Path` is empty on Windows, so the previous if does not apply and falls onto the error on the last condition. It gets detected if you prefix with file:// but that seems not friendly.

Comment (jszwedko) [inline altsrc/yaml_file_loader.go]: Aha, interesting, do you mind leaving a comment to that effect? That isn't immediately obvious looking at this code.

Comment (fernandezvara) [inline altsrc/yaml_file_loader.go]: For sure. That behaviour is strange, and of course, that `if` without a comment does not help understanding.

**Comment by jszwedko** [comment]
Thanks again!

**Pull request merged**